app/Repositories/SignupRepository.php

```diff
@@ -15,13 +15,15 @@ class SignupRepository
      */
     public function create($data)
     {
+        // Create the post_signup event
         $event = Event::create([
             'northstar_id' => $data['northstar_id'],
-            'event_type' => 'signup',
+            'event_type' => 'post_signup',
```

Comment:
Also when we create this signup, I think we should also store the other elements like quantity and why participated in the event record, right?